src/Reveal.Sdk.Dom.Tests/Visualizations/SingleValueLabelsVisualizationBaseFixture.cs

```diff
@@ -106,6 +153,13 @@ public void GetValues_ReturnsEmptyList_IfDataSpecIsNotSingleValueLabelsVisDataSp
             Assert.Empty(values);
         }
 
+        private class TestDimensionDataField : DimensionDataField
```

Comment:
instead of creating a new class, you could just use the TextDataField, unless there is a specific reason you need a new class.